v1/api/swagger/components/schemas/Project.yaml

```
@@ -40,4 +40,7 @@ allOf:
           - 91eec8b5-6460-4532-aad1-47b983edb63c
       VolumeReplicationEnabled:
         type: boolean
-        description: Indicates whether volume replication is enabled for this Project.
\ No newline at end of file
+        description: Indicates whether volume replication is enabled for this Project.
+      ParentID:
+        type: string
+        description: If not empty, ID of the Parent Project. Applicable to GreenLake Platform (GLP) only.
```

```suggestion
        description: ID of the Parent Project (optional). Applicable to GreenLake Platform (GLP) only.
```
would this be clearer?